src/create-job.ts

```diff
@@ -111,15 +114,24 @@ export type JobDescription = {
  * @param description An object that describes the job to create.
  */
 export default async function createJob(description: JobDescription, pipelineData?: any, encodingProfileData?: any, concurrency: boolean = true) {
-  logger.info(`Creating job ${description.name}.`);
+  logger.info(`Creating job ${description.name}`);
 
-  let pipeline: Pipeline | undefined = undefined;
+  let pipeline: any | undefined = undefined;
```

Comment:
Shouldn't this be of type `Pipeline`?

Reply:
If set to pipeline, Typescript does not identify class function calls of the class type EncorePipeline.

I have also tried Pipeline | EncorePipeline | undefined = undefined; This also causes errors.

Maybe there is an easy fix, I am not very familiar with Typescript.

Reply:
Resolves with [comment](https://github.com/Eyevinn/autovmaf/pull/58/files#r1365242726)